install.sh

```diff
@@ -329,7 +327,9 @@ nvm_do_install() {
   local PROFILE_INSTALL_DIR
   PROFILE_INSTALL_DIR="$(nvm_install_dir | command sed "s:^$HOME:\$HOME:")"
 
-  SOURCE_STR="\\nexport NVM_DIR=\"${PROFILE_INSTALL_DIR}\"\\n[ -s \"\$NVM_DIR/nvm.sh\" ] && \\. \"\$NVM_DIR/nvm.sh\"  # This loads nvm\\n"
+  SOURCE_STR="\\nexport NVM_DIR=\"${PROFILE_INSTALL_DIR}\"\\n[ -s \"\$NVM_DIR/nvm.sh\" ] && \\. \"\$NVM_DIR/nvm.sh\"  # This loads nvm\\n\n"
+  ZSH_COMPLETION_STR='# Load compinit/compdef\n autoload -U compinit\n compinit\n\n'
```

Comment:
is there a reason that this line needs to go in `install.sh`, instead of checking `ZSH_VERSION` inside the `bash_completion` file?

Reply:
Actually no, so I moved it to bash completion. This helps clean up a lot of unnecessary lines.